src/main/convert.js

```diff
@@ -218,7 +231,8 @@ export class ThriftFileConverter {
   ${values}
 |}>  = Object.freeze({
```

Comment:
This looks like this code is already here for generating enum maps. What is the point of duplicating it?